runtime/src/accounts_db.rs

```diff
@@ -148,11 +148,17 @@ pub struct AccountsAddRootTiming {
     pub store_us: u64,
 }
 
+#[derive(Debug, Default, Clone)]
+pub struct FillerAccountsConfig {
+    pub count: Option<usize>,
+    pub size: Option<usize>,
+}
+
 #[derive(Debug, Default, Clone)]
 pub struct AccountsDbConfig {
     pub index: Option<AccountsIndexConfig>,
     pub accounts_hash_cache_path: Option<PathBuf>,
-    pub filler_account_count: Option<usize>,
+    pub filler_accounts: Option<FillerAccountsConfig>,
```

Comment:
1. Please rename to `filler_accounts_config`.
2. Since the filler accounts configuration is so simple (and we can always check the filler accounts config `count`), can remove the `Option`.